script/toy/MangroveJs.s.sol

```diff
@@ -58,7 +61,11 @@ contract MangroveJsDeploy is Deployer {
       symbol: "TokenA",
       _decimals: 18
     });
+
+    broadcast();
+    tokenA.setMintLimit(type(uint).max);
     fork.set("TokenA", address(tokenA));
+
     broadcast();
     tokenA.setMintLimit(type(uint).max);
```

Comment:
This seems like a merge error? No need to set mint limit twice, right?

Same for Token B below.